nvchecker/source/github.py

```diff
@@ -5,6 +5,29 @@
 from pkg_resources import parse_version
 from tornado.httpclient import AsyncHTTPClient, HTTPRequest
 
+try:
+  import pyalpm
+  def cmp_to_key(mycmp):
+    'Convert a cmp= function into a key= function'
+    class K(object):
+      def __init__(self, obj, *args):
+        self.obj = obj
+      def __lt__(self, other):
+        return mycmp(self.obj, other.obj) < 0
+      def __gt__(self, other):
+        return mycmp(self.obj, other.obj) > 0
+      def __eq__(self, other):
+        return mycmp(self.obj, other.obj) == 0
+      def __le__(self, other):
+        return mycmp(self.obj, other.obj) <= 0
+      def __ge__(self, other):
+        return mycmp(self.obj, other.obj) >= 0
+      def __ne__(self, other):
+        return mycmp(self.obj, other.obj) != 0
+  vercmp = cmp_to_key(pyalpm.vercmp)
+except ImportError:
+  vercmp = None
```

Comment:
If it's not available, we should give errors, or users may get unexpected results.